en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+- The search API in Choreo then makes **1 call to the Flights Database** to retrieve the information according to the search criteria entered by the client. This is counted as **1 step**. 
+
+- **Assuming the entire [execution time](#executionTime) for the search process takes less than 500ms** (starting from making a call to Choreo, and Choreo making a call to the database), the search process thereby results in a **total of 2 steps**.
+
+<br/>
+
+#### Reserving a flight
+
+![Reserving a flight](../assets/img/references/rest-api-example2.png)
+
+Let’s see how steps are calculated for the reservation process:
+
+- As illustrated above, the client initiates the reservation process through the Web Application.
+- The Web Application proceeds to make **1 API call to the [Reservations API](#reservationsAPI)**. This is counted as 1 step.
+- The Reservation API then makes **1 API call to the [Bookings API](#bookingsAPI)**. This is counted as **1 step**. 
+- The bookings API makes **1 API call to the Booking Database** to update it with the booking details. This is counted as **1 step**.
+- Upon receiving the booking confirmation, the Reservation API makes **1 call to the [Twilio API](#twillioAPI)** to send an SMS confirmation to the customer. This is counted as **1 step**.
+
+- **Assuming the entire execution time for the reservation process takes 1200ms (1.2s)** , in addition to the above steps,  the total number of extra computation units consumed can be calculated as follows using ***floor((t)/500)) where t is the execution time***: 
+
+          floor(1200/500) = 2 units = 2 steps
+
+Therefore the total number of steps for the reservation process is: 
+
+           4 steps + 2 steps = 6 steps
+
+We can calculate the **total number of steps for searching and reserving a flight** as follows:
+
+- **Search process**: Single execution of a search would cost a total of **2 steps**. 
+- **Reservation process**: Single execution of reservation process would cost a total of **6 steps**. 
+- **Total**: A single execution of the search and reservation process of a flight would cost an organization a total of **8 steps in total**.
+
+!!! note
+    The above cost calculation is dependent on the above architecture and design of the APIs.
+<br/>
+
+### Example 2: REST API Proxy
+An organization exposes a flight tracking API as a managed API on Choreo. This system involves several APIs and databases. The scenario and the APIs in the system are described below:
+
+#### Scenario
+
+To search for real-time flight data, a client will enter search criteria, and search through the web application. Upon receiving the request, the client application (web application) will make a call to the [Flights Status API](#flightStatusAPI), a managed API on Choreo, to search for flight data. Choreo will then invoke the backend API, the [Real-Time Flight API](#realtimeFlightAPI). The backend API will invoke a database to get the information and pass it to the client.
```

Comment:
Present tense would be more appropriate here.

Reply:
done